spec/send_reset_email_spec.rb

```diff
@@ -0,0 +1,16 @@
+describe SendResetEmail do
+  let(:user) { double :user, password_token: 'qwertyuiop12fe23',
```

Comment:
Style/Blocks: Avoid using {...} for multi-line blocks.<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.